sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -28,17 +28,20 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
-#if ENABLE_DEBUG && defined(MODULE_IPV6_ADDR)
```

Comment:
unrelated, but also unnecessary as rpl does not build without `MODULE_IPV6_ADDR`. Do you want me to open a separate PR for this change?